src/lib/healthCheck/subServiceHealth.js

```diff
@@ -29,19 +29,24 @@ const Logger = require('@mojaloop/central-services-logger')
 
 const Config = require('../../lib/config')
 const Notification = require('../../handlers/notification')
+const Producer = require('@mojaloop/central-services-stream').Util.Producer
 const axios = require('axios')
 
 /**
  * @function getSubServiceHealthBroker
  *
  * @description Gets the health for the Notification broker
- * @returns Promise<SubServiceHealth> The SubService health object for the broker
+ * @returns Promise<object> The SubService health object for the broker
  */
 const getSubServiceHealthBroker = async () => {
   // TODO: Include use-case when running in API mode only (handlers.disabled=true) once 'getMetadata' enhancement has been added to the central-services-stream Producer.
   let status = statusEnum.OK
   try {
-    await Notification.isConnected()
```

Comment:
This doesn't seem to cater for when the ml-api is both a consumer and producer?

Reply:
For this we would need new service names 
we can't just use broker. should we start naming them consumers and producers?
we are going to have to cater for the also in central-ledger as you would have multiple consumers and producers

Reply:
I have made the appropriate changes @mdebarros, please re-review